components/create-collective/sections/CreateCollectiveForm.js

```diff
@@ -0,0 +1,238 @@
+    this.handleChange = this.handleChange.bind(this);
+
+    const collective = { ...(props.collective || {}) };
+    collective.slug = collective.slug ? collective.slug.replace(/.*\//, '') : '';
+
+    collective.backgroundImage = collective.backgroundImage || defaultBackgroundImage[collective.type];
+
+    this.state = {
+      collective,
+      checked: false,
+    };
+
+    this.categories = get(props.host, 'settings.apply.categories') || [];
+
+    if (this.categories.length === 1) {
+      this.state.collective.category = this.categories[0];
+    }
+
+    this.messages = defineMessages({
+      'tos.label': {
+        id: 'collective.tos.label',
+        defaultMessage: 'Terms of Service',
+      },
+    });
+  }
+
+  handleChange(fieldname, value) {
+    const collective = {};
+
+    collective[fieldname] = value;
+
+    this.setState({
+      collective: assign(this.state.collective, collective),
+    });
+  }
+
+  render() {
+    const { host, intl } = this.props;
+
+    const showForm = Boolean(this.categories.length === 0 || this.state.collective.category);
+
+    const initialValues = {
+      collectiveName: '',
+      collectiveDesc: '',
+      collectiveUrl: '',
+    };
+
+    const validate = values => {
+      const errors = {};
+
+      if (values.collectiveName.length > 10) {
+        errors.collectiveName = 'Please use fewer than 10 characters';
+      }
+
+      if (values.collectiveUrl.length < 5) {
+        errors.collectiveUrl = 'Please use more than 5 characters';
+      }
+
+      if (!values.collectiveDesc) {
+        errors.collectiveDesc = 'You need a description.';
+      }
+
+      return errors;
+    };
+
+    const submit = values => {
+      const { collectiveDesc, collectiveName, collectiveUrl } = values;
+      const collective = {
+        name: collectiveName,
+        description: collectiveDesc,
+        website: collectiveUrl,
+      };
+      assign(collective, this.state.collective);
+      this.props.onSubmit(collective);
+    };
+
+    return (
+      <div className="CreateCollectiveForm">
+        <div className="FormInputs">
+          {this.categories.length > 1 && (
+            <CollectiveCategoryPicker onChange={value => this.handleChange('category', value)} />
+          )}
+
+          {showForm && (
+            <div className="Form">
+              <Box p={4} m={4} style={{ border: '1px solid lightgrey', borderRadius: 8 }}>
+                <Formik
+                  validateOnChange={false}
+                  validateOnBlur={true}
+                  validate={validate}
+                  initialValues={initialValues}
+                  onSubmit={submit}
+                >
+                  {formik => {
+                    const { values, handleChange, handleSubmit, errors } = formik;
+
+                    return (
+                      <Form>
+                        <StyledInputField
+                          id="collectiveName"
+                          name="collectiveName"
```

Comment:
We're only working with collective values here, so no need to prefix

```suggestion
                          name="name"
```